pkgs/development/tools/cfn_flip/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "cfn_flip";
```

```suggestion
  pname = "cfn-flip";
```
You will need to specify `pname = "dfn_flip"` inside `fetchPypi`.